### PR TITLE
Syntax Sql correction and minor corrections

### PR DESCRIPTION
**Pull request by AngelDiaz591**
Minimal correction of the syntax of the procedure: `/reactions/procedures.sql`
Changing the `$$` by `;` to be able to execute all the procedures at the same time without having to do it one by one.
A small test was made and based on the result of that test we added what was missing, only a few things were added.
Pages where changes were made:
image
image

### CODE REVIEW

File: views/posts/index.php [hunk 1]
```diff
@@ -44,12 +44,16 @@
...
           <div class="actions">
             <div class="info">
-              <p class="likes <?= isset($_SESSION['user']) ? '' : 'openModal' ?>" id="reactions-count-<?= $d['id'] ?>">
-                <img src="/resources/img/like.png" alt="like"> <?= $d['total_reactions'] ?> reactions 
-              </p>
+              <a href="/posts/show/id:<?= $d["id"]; ?>">
+                <p class="likes <?= isset($_SESSION['user']) ? '' : 'openModal' ?>" id="reactions-count-<?= $d['id'] ?>">
+                  <img src="/resources/img/like.png" alt="like"> <?= $d['total_reactions'] ?> reactions 
+                </p>
+              </a>
```
Comment (Zama15) [inline views/posts/index.php:51]: this do not interfere with the click made to create the reaction? i mean if i click on the lick button what will happen? open the modal to react or redirect to the /posts/show action?

**Review by Zama15 — APPROVED** [review]

File: views/posts/index.php (hunk 1 above)
Comment (IsmaelMontiel710) [reply views/posts/index.php:51]: The modal does not work, it was left because I thought what work to future, but teams of design to added the modal

Comment (AngelDiaz591) [reply views/posts/index.php:51]: not knowing that this section was going to be made, I decided to make the change I did, so, is everything ok?

Comment (IsmaelMontiel710) [reply views/posts/index.php:51]: because, you make a redirection in the reactions information, you can put it outside the actions class

**Review by IsmaelMontiel710 — CHANGES_REQUESTED** [review]

**Pull request merged**